test/test1.conf

```diff
@@ -71,7 +74,7 @@ interface eth0 {
 	abro fe80::a200:0:0:1/64 {
 		AdvVersionLow 10;
 		AdvVersionHigh 2;
-		AdvValidLifetime 2;
+		AdvValidLifeTime 2;
```

Comment:
Changing the `...Lifetime` into `...LifeTime` (time with T) feels wrong when seeing `AdvRDNSSLifetime` (time with t).

For the record: I will approve this merge request so the radvd project  can move on.